Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM pytorch/pytorch:2.3.1-cuda11.8-cudnn8-runtime
+FROM pytorch/pytorch:2.4.1-cuda11.8-cudnn9-runtime
```

Comment:
Could it work with pytorch:2.5.1 too ? 
https://hub.docker.com/layers/pytorch/pytorch/2.5.1-cuda11.8-cudnn9-runtime/images/sha256-7aac344854fbc920da85f9abccb8e397a5bf99445553df9f4dfbde18009f4cd3?context=explore